src/main/java/org/generationcp/middleware/dao/RoleDAO.java

```diff
@@ -58,4 +61,32 @@ public Role getRoleById(final Integer id) {
 		return role;
 	}
 
+	public long countRolesUsers(final RoleSearchDto roleSearchDto) {
```

Comment:
@cuenyad this name should be countRoles only? I dont see anything related to users here

Reply:
done!